Advisor/Network/NetworkManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  request.swift
+//  Advisor
+//
+//  Created by Ilgam Akhmatdionov on 30.09.2021.
+//
+
+import Foundation
+
+class NetworkManager {
```

Comment:
why for network you use manager but for location service? 

https://stackoverflow.com/questions/8403535/when-do-i-call-my-class-controller-manager-or-service